_docs/configuration_files/foremast_config.rst

```diff
@@ -141,6 +141,12 @@ security group name, it will self-reference to its own application name.
     | *Required*: No
     | *Example*: ``{ "bastion" : [ { "start_port": "22", "end_port": "22", "protocol": "tcp" } ] }``
 
+``ec2_pipeline_types``
+**********************
+
+.. autodata:: foremast.consts.EC2_PIPELINE_TYPES
```

Comment:
So why not simply add the content here as the rest of the docs?

Reply:
I've been trying to get us to move the documentation into the code for a while. This way, if the code needs to change, the documentation is right there to update. In particular with the configuration documentation, the defaults are more likely to match the code reality. The secondary benefit is the `automodule` generated documentation also has the information when looking at the source documentation.